rocAL/rocAL/include/decoders/video/video_decoder.h

```diff
@@ -44,20 +44,6 @@ extern "C"
 #endif
 #include "parameter_factory.h"
 
-enum class VideoDecoderType
```

Comment:
I see this enum has been moved to image, is this intended ?

Reply:
Yes it is intentionally removed and cases are added in DecoderType